src/utils/fiatToCrypto.ts

```diff
@@ -87,19 +90,43 @@ export const wertWidgetUrl = (address: string, fiatValue: string) => {
   return wertWidget.getEmbedUrl();
 };
 
+export const pelerinSupportedAssets = [
+  'BNB',
+  'BTCB',
+  'BUSD',
+  'DAI',
+  'ETH',
+  'FRAX',
+  'LUSD',
+  'MATIC',
+  'USDC',
+  'USDT',
+  'WBTC',
+  'WETH',
+  'XDAI',
+];
```

Comment:
This might be something we'll need to move to Firebase Remote Config as this list tends to change over time. When you get around to this, please ping me.

Reply:
Firebase remote config paramter created: `app_mt_pelerin_supported_assets `